brew/README.md

```diff
@@ -0,0 +1,19 @@
+## Instructions for installing the KubeStellar user commands and kubectl plugins using Brew package manager
+
+install brew:
+    https://brew.sh
+
+to install:
+    HOMEBREW_NO_INSTALL_FROM_API=1 brew install -s --formula kcp_cli.rb
+    HOMEBREW_NO_INSTALL_FROM_API=1 brew install -s --formula kubestellar_cli.rb
+
+to remove: 
+    brew remove kubestellar_cli
+    brew remove kcp_cli
+
+
+after PR is accepted:
+
+brew tap kubestellar/kubestellar https://github.com/kubestellar/kubestellar/brew
```

Comment:
What is going to make this happen?  When I put `https://github.com/kubestellar/kubestellar/bootstrap` in the address bar of my browser, I get a 404.

Reply:
This looks like #1153

Reply:
going to update this - I do not think we need the url at the end.